tests/ciscotest.rb

```diff
@@ -62,14 +65,40 @@ def config(*args)
     result
   end
 
+  def ip_address?(ip)
+    return true if IPAddr.new(ip).ipv4?
```

Comment:
Does this properly return false for IPv6? I would have expected this to be written as `return IPAddr.new(ip).ipv4?`